install.sh

```diff
@@ -56,38 +56,45 @@ nvm_download() {
 }
 
 install_nvm_from_git() {
-  if [ -d "$NVM_DIR/.git" ]; then
-    echo "=> nvm is already installed in $NVM_DIR, trying to update"
+  local install_dir
+  install_dir=$(nvm_install_dir)
+  if [ -z "$NVM_SOURCE" ]; then
+    NVM_SOURCE="https://github.com/creationix/nvm.git"
+  fi
+
+  if [ -d "$install_dir/.git" ]; then
+    echo "=> nvm is already installed in $install_dir, trying to update"
     printf "\r=> "
-    cd "$NVM_DIR" && (command git fetch 2> /dev/null || {
-      echo >&2 "Failed to update nvm, run 'git fetch' in $NVM_DIR yourself." && exit 1
+    cd "$install_dir" && (git fetch 2> /dev/null || {
```

Comment:
I recently added `command` to ensure that `git` isn't aliased - can you ensure that all `git` commands are prefixed with `command`?